core/src/components/button/button.ios.scss

```diff
@@ -212,3 +212,12 @@
 :host(.button-solid.ion-color.ion-activated) .button-native::after {
   background: #{current-color(shade)};
 }
+
+
+// Activated Button in Toolbar
+// --------------------------------------------------
+
+:host(.button-outline.ion-activated.in-toolbar:not(.ion-color):not(.in-toolbar-color)) .button-native {
+  background: var(--ion-toolbar-color, var(--color));
```

Comment:
Why are we using color variables for background?

Reply:
The background and text colors reverse when an outline button is activated. Note that the iOS styles have not changed from `main`: https://github.com/ionic-team/ionic-framework/pull/29216/files#diff-5b2387a45e4270ba71b3a393ba3fe681d917cc518e6f9e059cccdcbe56dd2149L339